pdaltools/_version.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "1.7.3"
+__version__ = "1.7.4"
 
```

Comment:
Pour plus de lisibilité de l'historique des commits, quand tu changes de version
- il faut faire un commit à part
- et mettre le n° de version final dans le message de commit, "bump to version 1.7.4"

Reply:
retour à .3
On mettre le 4 au dernier moment